src/common/FastRational.h

```diff
@@ -674,6 +745,7 @@ inline void substraction(FastRational& dst, const FastRational& a, const FastRat
             dst.den = zd;
         }
         dst.kill_mpq();
+        dst.setWordPartValid();
```

Comment:
I think we agreed that this is not necessary, `kill_mpq` takes care of that. It is not present in the `addition`.

Reply:
Right.  I spotted a few others as well and fixed them.